magefile.go

```diff
@@ -149,8 +149,3 @@ func CheckLicenseHeaders() error {
 func DumpVariables() error {
 	return devtools.DumpVariables()
 }
-
-// Vendor moves dependencies to the repo using go modules.
-func Vendor() error {
-	return devtools.Vendor()
-}
```

Comment:
This target is used in `libbeat/scripts/Makefile`, it should be removed from there.